### PR TITLE
updating license

### PR DESCRIPTION
**Pull request by wkozmaNTIA**
Updating the text of the license file

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+SOFTWARE DISCLAIMER / RELEASE
 
-You can use, copy, modify, and redistribute the NTIA/ITS developed software upon your acceptance of these terms and conditions and upon your express agreement to provide appropriate acknowledgments of NTIA's ownership of and development of the software by keeping this exact text present in any copied or derivative works.
+This software was developed by employees of the National Telecommunications and Information Administration, an agency of the Federal Government and is provided to you as a public service.  Pursuant to title 15 United States Code Section 105, works of NTIA employees are not subject to copyright protection within the United States.
```
Comment (jhazentia) [inline LICENSE.md]: Should NTIA be put in parenthesis after National Telecommunications and Information Administration? For example, "developed by employees of the National Telecommunications and Information Administration (NTIA)," 

Should "title" be capitalized?  For example, "Title 15 United States Code Section 105".

Comment (MichaelCotton) [reply LICENSE.md]: Yes, to both.

Comment (wkozmaNTIA) [reply LICENSE.md]: Changes applied

**Comment by jhazentia** [comment]
LGTM

**Pull request merged**